3723/1/pages.rb

```diff
@@ -0,0 +1,63 @@
+require_relative 'repo'
+require_relative 'taker'
+require 'terminal-table'
+require 'nokogiri'
+require 'open-uri'
+
+class Pages
```

Comment:
Another unclear name, the same question, is there a way to make it better?

Reply:
This class does many different things, it's hard to read. If you leave only one public method here and it performs all the stuff, it would describe that it follows single responsibility principle from the first sight. If it's impossible, you better think of decomposition of the class into a few ones

Reply:
Done